### DISCUSSION TITLE
1.4.5

### DISCUSSION THREAD

**Discussion by ivan-aksamentov** [nextstrain/nextclade#601] [Discussion/Chat]
## Nextclade Web 1.7.3, Nextclade CLI 1.4.5 (2021-11-04)

This is a bugfix release for Nextclade Web and Nextclade CLI.

After Nextclade Web version 1.7.3 and Nextclade CLI version 1.4.3 the "Private mutations" QC check was incorrectly counting individual deletions, rather than contiguous ranges of deletions, as it was before that. This resulted in QC score being too high for sequences that have many deletions. In this version we ensure again that the deletion ranges are counted. This should now bring the "Private mutations" QC score back to what is expected.

<hr /><em>This discussion was created from the release <a href='https://github.com/nextstrain/nextclade/releases/tag/1.4.5'>1.4.5</a>.</em>

**Comment by corneliusroemer** [comment]
It would be good if we had tests that checked whether any qc scores go totally bonkers after an update. This is hard, because expected results could change with new features but it would help us spot such issues before release?
Basically we could provide test sequences and flag if they change. If they change, we have to change the test if it's desired or fix the problem if regression.

**Reply by ivan-aksamentov** [reply]
Yes, I have e2e tests setup, just need to make them more convenient to run and to automate
https://github.com/nextstrain/nextclade/blob/master/e2e/cli/README.md
